targets/rpi3.json

```
@@ -0,0 +1,13 @@
+{
+	"inherits": ["arm64"],
+	"build-tags": ["rpi3", "rpi", "baremetal"],
+	"cflags": [
+		"-mcpu=cortex-a53",
+		"-march=armv8-a"
+	],
+	"linkerscript": "targets/rpi3.ld",
```

As you mentioned in Slack, this line is unused. Instead, pass the linker script to the linker this way:

```suggestion
	"ldflags": [
		"-T", "targets/rpi3.ld"
	],
```